app.rb

```diff
@@ -0,0 +1,150 @@
+require 'date'
+require_relative 'person_manager'
+require_relative 'book_manager'
+require_relative 'rental_manager'
+
+# app.rb
+class App
+  def initialize
+    @person_manager = PersonManager.new
+    @book_manager = BookManager.new
+    @rental_manager = RentalManager.new
+  end
+
+  def exit_app
+    puts 'Exiting the Library App. Goodbye!'
+    exit
+  end
+
+  def run
+    display_welcome_message
+
+    loop do
+      print_options_menu
+      option = gets.chomp.to_i
+      process_option(option)
+      break if exit_option?(option)
+    end
+
+    display_goodbye_message
+  end
+
+  private
+
+  def display_welcome_message
+    puts ''
+    puts 'Welcome to the Library App!'
+  end
+
+  def print_options_menu
+    puts ''
+    print 'Please choose an option by selecting a number: '
+    puts "\nOptions:"
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List rentals for a person with a given id'
+    puts '7. Exit'
+  end
+
+  def exit_option?(option)
+    option == 7
+  end
```

Comment:
- [ ] As stated in the project requirement below, this part of the code should be placed in the `main.rb` file which is to serve as the entry point to your application please kindly make this correction :+1: 
![main](https://github.com/enocol/old_school_library/assets/89055502/58beb400-7b41-464f-a69d-5ebb1ff41a71)